reference/configuration/framework.rst

```diff
@@ -1955,6 +1955,58 @@ A list of cache pools to be created by the framework extension.
 
     For more information about how pools works, see :ref:`cache pools <component-cache-cache-pools>`.
 
+To configure a redis cache pool with a default lifetime of 1 hour, do the following:
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # config/packages/framework.yaml
+        framework:
+            cache:
+                pools:
+                    cache.mycache:
+                        adapter: cache.adapter.redis
```

Comment:
how would this name follow from the rest of this documentation page?

Reply:
@nicolas-grekas how to know the right name here, where does it come from? in default_redis_provider we say "The provider is available as the ``cache.redis``
service." but here we need the cache.adapter.redis - is there always a cache.adapter.X for each cache.X ?

Reply:
thanks! i updated the documentation of the `adapter` setting with this information. i mention the `cache.pool` tag, but if you prefer we can also remove that.